src/Container.ts

```diff
@@ -49,6 +50,10 @@ export default abstract class Container extends Component implements IContainer
                     continue;
                 }
                 const component = new componentClass(this.main, this, opts);
+                if (component instanceof Inert) {
```

Comment:
Sorry If I wasn't clear about this in my previous comment. We'll never have an `Inert` component in the preset so this if block will never run. What we really want to do here is to implicitly create `Inert` component when `componentClass` cannot be found in the registry. So we can just replace the `continue;` in the previous if block with something like `componentClass = Inert;`.